app/services/onlinetesting/OnlineTestService.scala

```diff
@@ -77,10 +77,8 @@ trait OnlineTestService {
 
   def getPhase1TestProfile(userId: String): Future[Option[Phase1TestProfile]] = {
     appRepository.findCandidateByUserId(userId).flatMap {
-      case Some(candidate) if candidate.applicationId.isDefined =>
-        otRepository.getPhase1TestProfile(candidate.applicationId.get)
+      case Some(candidate) => otRepository.getPhase1TestProfile(candidate.applicationId.getOrElse(""))
```

Comment:
Under what circumstance would you want to get a getPhase1TestProfile if the candidate didn't actually have an application? If there isn't any circumstance that make sense, it seems to me that a plain old .get is better here, or the guard.